docs/testing/component-tests.md

```diff
@@ -1,82 +1,89 @@
 # Component Tests
 
-Calypso has a lot of React UI components. (Try for example running `find -name *.jsx` under the `client` folder). This combined with the fact that UI tests are among the more difficult things to (unit) test, a guide or checklist for how to make it as easy and focused as possible seems appropriate.
+Calypso has a lot of React UI components. (Try for example running `find . -name '*.js?'` from the `client` folder).  It can be difficult to test components and UI. This guide will help make it as easy and focused as possible.
 
 ## [Getting started](#getting-started)
 
-To run all current client side tests, run `npm run test-client` in the root source folder. You can also run individual tests. Check [Testing overview](testing-overview.md#client-side-tests) documentation for more details.
+To run all current client side tests, run `npm run test-client` from within the Calypso source. You can also run individual tests, see [Testing overview](testing-overview.md#client-side-tests) for more details.
```

Comment:
Probably the most common use case is a single test? If so could be better to link to  *How to run specified suite or test-case*.